docs/roman/pipeline_run.rst

```diff
@@ -95,13 +94,11 @@ For the mosaic level pipeline and steps,
 ::
 
  from romancal.pipeline import MosaicPipeline
- result = ExposurePipeline.call('r0000101001001001001_asn.json')
+ mosp = MosaicPipeline('r0000101001001001001_asn.json')
```

Comment:
```suggestion
result = MosaicPipeline.call('r0000101001001001001_asn.json')
```
Thanks for switching this to `MosaicPipeline`. I think from the text above that this should be a `call` as it's describing the pipeline results.